client/src/services/ApiService.js

```diff
@@ -51,6 +51,8 @@ class CustomHttpClient {
 
   sendRequest(httpRequest) {
     const url = new URL(httpRequest.url);
+    url.searchParams.delete("api-version");
+    url.searchParams.append("api-version", API_VERSION);
```

Comment:
Would `set` be a better option here? https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/set

Reply:
Changed into set and did a quick test, sorry I didn't saw it in the original example I found as an option